src/dotnet/Audio.UI.Blazor/Components/AudioPlayer/worklets/feeder-audio-worklet-processor.ts

```diff
@@ -85,12 +86,14 @@ class FeederAudioWorkletProcessor extends AudioWorkletProcessor implements Feede
         return ResolvedPromise.Void;
     }
 
-    public resume(_noWait?: RpcNoWait): Promise<void> {
-        if (this.playbackState !== 'paused')
+    public resume(): Promise<void> {
+        if (this.playbackState === 'playing')
             return;
 
         debugLog?.log(`#${this.id}.resume`);
-        this.playbackState = 'playing';
+        this.playbackState = this.playbackState === 'ended'
```

Comment:
Why do you switch playbackstate from ended to paused here?

Reply:
Because initial state is `private playbackState: PlaybackState = 'paused';`